ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+# Version 0.2.2 - 2022-06-21
+* Add support for enabling internal caching to improve performance.
+* Fixing a bug where fields of type Dict[...] were not being parsed properly.
```

Comment:
Can we split this bugfix into its own commit?

Reply:
Yes, I can do that.